azurerm/internal/services/eventhub/eventhub_namespace_resource.go

```diff
@@ -79,6 +79,12 @@ func resourceArmEventHubNamespace() *schema.Resource {
 				Default:  false,
 			},
 
+			"zone_redundant": {
```

Comment:
running the tests for this we also need to set this into the state during the Read function, otherwise the test fails with:

```
------- Stdout: -------
=== RUN   TestAccAzureRMEventHubNamespace_basic
=== PAUSE TestAccAzureRMEventHubNamespace_basic
=== CONT  TestAccAzureRMEventHubNamespace_basic
TestAccAzureRMEventHubNamespace_basic: testing.go:640: Step 1 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
(map[string]string) {
}
(map[string]string) (len=1) {
(string) (len=14) "zone_redundant": (string) (len=5) "false"
}
--- FAIL: TestAccAzureRMEventHubNamespace_basic (237.43s)
FAIL
```

could we update this? this should be adding `d.Set("zone_redundant", props.ZoneRedundant)` in the Read function